cpp/examples/abm_minimal.cpp

```diff
@@ -145,6 +145,7 @@ int main()
         if (infection_state != mio::abm::InfectionState::Susceptible) {
             person.add_new_infection(mio::abm::Infection(mio::abm::VirusVariant::Wildtype, person.get_age(),
                                                          world.parameters, start_date, infection_state));
+        }
```

Comment:
Hi @khoanguyen-dev Thanks for addressing my comment. What however bothers me is that here only one bracket has been added (removed before). There then must have been an error before?!

Reply:
Hi @mknaranja. I left out the brackets due to some mixed up while merging with the main. 
I added back the **two brackets** after your first comment. My guess is the first test think it is a one-liner if statement so didn't raise any error.

Reply:
Hi @mknaranja. According to the [history of abm_minimal.cpp](https://github.com/DLR-SC/memilio/commits/763cde835fb6cc22a1a6bd5d3aa2eb1702ead23d/cpp/examples/abm_minimal.cpp), the brackets were not introduced before yesterday's merge with main. I remember removing some brackets while merging so probably why the wasn't update correctly. 
I still don't know why Github only detect one bracket was deleted. Maybe just the bracket on line 148 was somehow interpreted as the same position as before.